python_utils/rec.py

```diff
@@ -0,0 +1,95 @@
+import sys
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+
+perm_amp = []
+perm_phase = []
+# Create figure for plotting
+plt.ion()
+fig = plt.figure()
+ax = fig.add_subplot(111)
+count = 0
+plt.xlabel("X-axis")
+plt.ylabel("Y-axis")
+plt.title("Updating plot...")
+fig.canvas.draw()
+plt.show(block=False)
+process_line = True
+
+
+def carrier_plot(amp):
+
+    #Adds xlims after 100 entries
+    if count > 100:
+        plt.xticks([], [])
+        plt.xlim()
+
+    else:
+        plt.xlim(0, 100)
+    plt.xlabel("Packet number/Time")

```

Comment:
I think if xlabel, ylabel, and title are set at the beginning, it is not necessary to update it in every `carrier_plot` call.